tests/conftest.py

```diff
@@ -40,6 +35,8 @@ def _generate_app(
         app_data=None,
         watch=False,
     ):
+        from aiidalab.app import AiidaLabApp, _AiidaLabApp
```

Comment:
Why is the import moved here?

Reply:
I don't remember if I had a big reason for this TBH. This is test code so I don't care much either way.